app/decorators/models/solidus_easypost/spree/shipment_decorator.rb

```diff
@@ -30,12 +34,18 @@ def selected_easy_post_shipment_id
       end
 
       def build_easypost_shipment
-        ::EasyPost::Shipment.create(
+        shipment_hash = {
           to_address: order.ship_address.easypost_address,
           from_address: stock_location.easypost_address,
           parcel: to_package.easypost_parcel,
           options: ::Spree::Easypost::Config.ddp_enabled ? { incoterm: 'DDP' } : {}
-        )
```

Comment:
Whats this?  What does DDP mean in this context?

Reply:
Addresses the issue here: https://github.com/solidusio-contrib/solidus_easypost/issues/63